views/campaign-signup.ejs

```diff
@@ -0,0 +1,4 @@
+<!-- Re-using Partner Microsite styles -->
+<div class="partner-page">
```

Comment:
I'm going back and forth on this. Would it make sense to instead just use the partner-signup.ejs template? If it's using the same styles anyway, then I'd prefer we just use the already existing ejs template. Otherwise, this feels like it should have its own unique class name like `campaign-page`. Or the `partner-page` class should be renamed/refactored to be for more generic use.